Makefile

```diff
@@ -17,7 +17,7 @@ deps:
 fmt: sort-imports
 	@find . -name '*.go' -type f -not -path "*.git*" -not -name '*.pb.go' -not -name '*pb_test.go' | xargs gofmt -w -s
 	@find . -name '*.go' -type f -not -path "*.git*"  -not -name '*.pb.go' -not -name '*pb_test.go' | xargs goimports -w -local github.com/celestiaorg
-	@go mod tidy -compat=1.17
+	@go mod tidy -compat=1.20
```

Comment:
Not 1.21?

Reply:
also good point

Reply:
Then we can just remove the flag. It's for compatibility with older versions, iirc